rails/Dockerfile

```diff
@@ -1,10 +1,8 @@
 ARG NODE_VERSION=18.18.0
-ARG RUBY_VERSION=2.7
+ARG RUBY_VERSION=3.2
 
 FROM node:${NODE_VERSION}-alpine AS node
-# Stick to Alpine <= 3.13 to avoid issues with ECS Docker
-# See: https://github.com/docker-library/ruby/issues/351#issue-940462153
-FROM docker.io/ruby:${RUBY_VERSION}-alpine3.12
+FROM ruby:${RUBY_VERSION}-alpine3.18
```

Comment:
Do we know if the issue on ECS that caused us to pin to `alpine3.12` has been resolved?  I couldn't tell in the comments in the link that was deleted.